web/tls_config.go

```diff
@@ -51,6 +53,12 @@ type TLSStruct struct {
 	PreferServerCipherSuites bool       `yaml:"prefer_server_cipher_suites"`
 }
 
+type FlagStruct struct {
```

Comment:
Minor naming nit, we usually call this `Options` or `Config`. `FlagStruct` being a struct is a little redundant. :smile:

Reply:
I think I saw `TLSStruct`, so I called it that. It seems that `Config` is already taken (right above it), and `Options` doesn't really tell you what the difference is between `Options` and `Config` by the name. So, how about `FlagConfig`?

Reply:
Sounds fine.